index.html

```diff
@@ -0,0 +1,135 @@
+                    </div>
+                </div>
+                <span onclick="openNav()" class="mobile">&#9776;</span>
+            </div>
+            <nav class="nav">
+                <ul class="desktop">
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="about.html">About Us</a></li>
+                    <li><a href="rooms.html">Rooms</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+
+    <!-- header Section -->
+    <section class="home">
+        <div class="hero">
+            <h1>"We Welcome You All At"</h1>
+            <h2>Wills A Luxury Hotel</h2>
+            <p>At Wills Hotel, we accommodate people from the whole corners of the wolrd.Everyday we prepare to receive
+                and facilitate people from different cultures both locally and all over the globe.
+            </p>
+
+        </div>
+    </section>
+
+    <section class="background">
+        <h1>Our Rooms</h1>
+        <hr>
+        <div class="Room-section">
+            <div class="Room1">
+                <div class="price">
+                    <h2>Double Rooms</h2>
+                    <h3><span>199$</span>/Pernight</h3>
+                </div>
+            </div>
+            <div class="Room2">
+                <div class="price">
+                    <h2>Premium King Room</h2>
+                    <h3><span>159$</span>/Pernight</h3>
+                </div>
+            </div>
+            <div class="Room3">
+                <div class="price">
+                    <h2>Deluxe Room</h2>
+                    <h3><span>198$</span>/Pernight</h3>
+                </div>
+            </div>
+            <div class="Room4">
+                <div class="price">
+                    <h2>Family Room</h2>
+                    <h3><span>299$</span>/Pernight</h3>
+                </div>
+            </div>
+        </div>
+        <button type="button" class="morebtn"> See More Rooms</button>
+        <h2><a href="#" class="desktop-only">See More Rooms</a></h2>
+    </section>
+    <!-- Dynamic Section -->
+    <section class="services-section">
+        <div class="work">
+            <h1>Our Services</h1>
+            <hr>
+        </div>
+        <div class="services">
+
+        </div>
+
+        <button type="button" class="seeMore">see More</button>
+    </section>
+    <section class="sponser">
+        <h1>Our Partners</h1>
+        <hr>
+        <div class="sponser-image">
+            <a href=""><img src="https://promzzykoncepts.github.io/NFT-Conference/images/coinbase.svg" alt=""></a>
+            <a href=""><img src="https://promzzykoncepts.github.io/NFT-Conference/images/binance-Logo.png" alt=""></a>
+            <a href=""><img src="https://promzzykoncepts.github.io/NFT-Conference/images/Flow.png" alt=""></a>
+            <a href=""><img src="https://promzzykoncepts.github.io/NFT-Conference/images/moonpaymain.svg" alt=""></a>
+        </div>
+    </section>
+
+    <footer>
+
```

Comment:
- [ ] In the partner section for the desktop version I think you just need to make the elemtns width a little bit smaller 🔥🔥 to be identical to the original design ✅🔥

- Your design
![image](https://user-images.githubusercontent.com/58553711/181908067-6cd46f0c-dab4-4db1-b173-7fc405869fb3.png)

- Original design
![image](https://user-images.githubusercontent.com/58553711/181908084-5ac1fc36-1a14-4c8d-ac72-0579fcda6683.png)